docs/cheqd-cli/cheqd-cli-identity.md

```diff
@@ -0,0 +1,189 @@
+
+## Crypto commands in cheqd CLI - EXPERIMENTAL FUNCTIONALITY - Do not use in production!
+
+
+#### Command for generating public and private part of verification key
+
+```bash
+$ cheqd-noded debug ed25519 random >> keys.txt
+```
+
+P.S. it's very important to keep your private verification key in a safe place. 
+The simpliest suggestion here is to redirect output to file as shown in bash command above.
+
+#### Response example
+
+```text
+{"pub_key_base_64":"MnrTheU+vCrN3W+WMvcpBXYBG6D1HrN5usL1zS6W7/k=","pub_key_multibase_58":"",\
+"priv_key_base_64":"FxaJOy4HFoC2Enu1SizKtU0L+hmBRBAEpC+B4TopfQoyetOF5T68Ks3db5Yy9ykFdgEboPUes3m6wvXNLpbv+Q=="}
+```
+
+#### Convert base64 to multibase58
+```bash
+$ cheqd-noded debug encoding base64-multibase58 <public_key_in_base64_representaion>
+```
+
+#### Response example
+```text
+$ cheqd-noded debug encoding base64-multibase58 MnrTheU+vCrN3W+WMvcpBXYBG6D1HrN5usL1zS6W7/k=
+z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE
+```
+
+
+## Identity-related commands in cheqd CLI
+
+**Note**: The `--chain-id` and `--node` flags are optional and only required if using the CLI from a machine that is not running an active cheqd node on mainnet/testnet.
+
+### Create a new DID
+
+#### Command
+
+```bash
+$ cheqd-noded tx cheqd create-did <did-document-json> <did-verification-method-id> --ver-key <did-private-key> --from <wallet-name> --gas auto --gas-adjustment 1.2 --gas-prices 25ncheq --chain-id <chain-id> --node <node-rpc-endpoint>
+```
+
+#### Sample DID Document JSON
+
+```json
+{
+  "id": "did:cheqd:<namespace>:<unique-id>",
+  "verification_method": [
+    {
+      "id": "did:cheqd:<namespace>:<unique-id>#<key-alias>",
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:cheqd:<namespace>:<unique-id>",
+      "public_key_multibase": "<verification-public-key-multibase>"
+    }
+  ],
+  "authentication": [
+    "did:cheqd:<namespace>:<unique-id>#<auth-key-alias>"
+  ]
+}
+```
+
+#### Arguments
+
+* `id`: A unique identifier for format `did:cheqd`, conforming to the [cheqd DID method specification](../../architecture/adr-list/adr-002-cheqd-did-method.md).
+
+* `id` feild for `verification_method` section: key identifier in the following format: `did:cheqd:<namespace>:<unique-id>#<key-alias>`. Exapmle: `did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1`.
+* `--ver-key`: Base64 encoded ed25519 private key to sign identity message with. A pair for the key from DID Document. \
+    Use for testing purposes only because the key will be stored in shell history!
+* `--from`: Cosmos account key which will pay fees for the transaction ordering.
+* `--node`: IP address or URL of node to send request to
+* `--chain-id`: i.e. `cheqd-testnet-4`
+* `--fees`: Maximum fee limit that is allowed for the transaction.
+
+
+#### Example
+
+```bash
+$ cheqd-noded tx cheqd create-did '{"id": "did:cheqd:testnet:zJ5EDiiiKWDyo79n",\
```

Comment:
We should change this example to similar to the tutorial where the cat based method is shown